addon/components/tooltip-on-parent.js

```diff
@@ -25,7 +25,9 @@ export default Ember.Component.extend({
         console.warn('No renderTooltip method found on the parent view of the {{tooltip-on-parent}} component');
       }
 
-      this.remove();
+      if (!this.get('isDestroyed') && !this.get('isDestroying')) {
```

Comment:
Yeah, I guess you're right. There can't be a situation where isDestroying=false and isDestroyed=true. I'll update the PR